api/v1/composition.go

```diff
@@ -160,3 +160,29 @@ func (c *Composition) InputsExist(syn *Synthesizer) bool {
 	}
 	return true
 }
+
+func (c *Composition) InputsMismatched() bool {
+	var minRevision *int
+	for _, rev := range c.Status.InputRevisions {
+		if rev.Revision == nil {
+			continue
+		}
+		if minRevision == nil {
+			minRevision = rev.Revision
+			continue
+		}
+		if *rev.Revision > *minRevision {
```

Comment:
`*rev.Revision > *minRevision`?

Reply:
I think this is right as-is, but `minRevision` should be named `maxRevision` :blush: